### PR TITLE
Add Cmore

### PR DESCRIPTION
**Pull request by slothtown**

### CODE REVIEW

File: img/c/cmore.se.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...0 55.132-25.35 55.132-56.607 0-31.012-24.858-56.61-55.132-56.61-30.52 0-55.132 25.598-55.132 56.61ZM748.216 171.302c0 8.86-4.923 14.276-14.522 14.276h-5.169c-9.598 0-14.767-5.415-14.767-14.276V25.35c0-9.107 5.169-14.276 14.767-14.276h55.379c30.765 0 55.623 25.352 55.623 55.378 0 17.967-10.583 34.211-25.596 43.81 8.614 7.383 14.028 16.245 17.474 27.567l8.122 28.058a32 32 0 0 1 1.723 8.614c0 6.398-5.906 11.076-13.538 11.076h-7.382c-7.63 0-12.06-3.693-14.03-10.585l-8.122-28.795c-3.444-12.06-14.028-24.613-31.503-24.613h-18.459Zm0-127.737v45.532h33.965c12.553 0 22.15-9.845 22.15-22.644 0-12.552-9.844-22.888-22.15-22.888ZM55.624 98.45c0 23.627 19.444 43.563 44.056 43.563 11.568 0 19.936-4.43 29.78-11.567 5.17-3.693 10.092-7.876 18.952-7.876a28.557 28.557 0 0 1 17.475 5.906c6.4 4.924 9.845 11.815 9.845 20.429 0 9.106-3.937 15.26-8.122 19.936-13.044 14.767-39.872 28.058-69.899 28.058C42.333 196.899 0 151.12 0 98.449S42.58 0 97.711 0c29.78 0 55.87 13.291 68.914 28.058 4.185 4.676 8.123 10.83 8.123 19.936a24.967 24.967 0 0 1-9.845 20.428 28.272 28.272 0 0 1-17.475 5.907c-8.86 0-13.783-4.183-18.951-7.876-8.861-6.4-16.983-11.568-29.29-11.568A43.804 43.804 0 0 0 55.625 98.45Z" class="a"/></svg>
```
Comment (hkamran80) [inline img/c/cmore.se.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 196.9"><path fill="#af3ce8" d="M382.23 118.39c-4.43 6.4-7.63 11.32-16.74 11.32h-4.67c-9.1 0-12.3-4.93-16.74-11.33L314.3 74.33v96.97c0 8.86-5.17 14.28-14.77 14.28h-4.92c-9.85 0-14.77-5.42-14.77-14.28V25.35c0-9.1 4.93-14.28 14.77-14.28h6.15c8.37 0 12.06 2.47 16.5 8.86l45.77 67.2 46.03-67.2c4.18-6.4 7.87-8.86 16.49-8.86h6.15c9.85 0 14.77 5.17 14.77 14.28V171.3c0 8.86-4.92 14.28-14.77 14.28h-4.92c-9.85 0-14.77-5.42-14.77-14.28V74.33Zm521.29 67.19c-9.6 0-14.77-5.42-14.77-14.28V25.35c0-9.6 5.66-14.28 14.77-14.28h80.73c9.1 0 14.76 4.68 14.76 14.28v3.94c0 9.6-5.65 14.27-14.76 14.27H923.2v38.89h50.7c9.1 0 14.77 4.68 14.77 14.28v4.18c0 9.35-5.66 14.03-14.77 14.03h-50.7v38.15h62.02c9.35 0 14.77 4.92 14.77 14.27v4.19c0 9.35-5.42 14.03-14.77 14.03ZM489.79 98.45c0-49.47 40.6-89.83 90.32-89.83s90.33 40.36 90.33 89.83-40.61 89.59-90.33 89.59-90.32-40.12-90.32-89.59Zm35.2 0c0 31.26 24.6 56.6 55.12 56.6 30.28 0 55.13-25.34 55.13-56.6 0-31.01-24.85-56.61-55.13-56.61-30.52 0-55.13 25.6-55.13 56.61Zm223.23 72.85c0 8.86-4.93 14.28-14.53 14.28h-5.17c-9.6 0-14.76-5.42-14.76-14.28V25.35c0-9.1 5.17-14.28 14.76-14.28h55.38c30.77 0 55.63 25.36 55.63 55.38 0 17.97-10.59 34.21-25.6 43.81 8.62 7.38 14.03 16.25 17.48 27.57l8.12 28.06a32 32 0 0 1 1.72 8.61c0 6.4-5.9 11.08-13.54 11.08h-7.38c-7.63 0-12.06-3.7-14.03-10.59l-8.12-28.8c-3.45-12.05-14.03-24.6-31.5-24.6h-18.46Zm0-127.74V89.1h33.96c12.55 0 22.15-9.85 22.15-22.65 0-12.55-9.84-22.89-22.15-22.89ZM55.62 98.45c0 23.63 19.45 43.56 44.06 43.56 11.57 0 19.94-4.43 29.78-11.56 5.17-3.7 10.1-7.88 18.95-7.88a28.56 28.56 0 0 1 17.48 5.9c6.4 4.93 9.84 11.82 9.84 20.43 0 9.11-3.93 15.26-8.12 19.94-13.04 14.77-39.87 28.06-69.9 28.06C42.33 196.9 0 151.12 0 98.45S42.58 0 97.71 0c29.78 0 55.87 13.3 68.92 28.06 4.18 4.67 8.12 10.83 8.12 19.93a24.97 24.97 0 0 1-9.85 20.43 28.27 28.27 0 0 1-17.47 5.9c-8.86 0-13.79-4.17-18.95-7.87-8.86-6.4-16.99-11.57-29.3-11.57a43.8 43.8 0 0 0-43.56 43.57Z"/></svg>
```

Here's a version that uses `viewBox`

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: img/c/cmore.se.svg (hunk 1 above)
Comment (slothtown) [reply img/c/cmore.se.svg]: Is viewBox preferred over width/height?

Comment (hkamran80) [reply img/c/cmore.se.svg]: Yes. Here's a [Stack Overflow answer](https://stackoverflow.com/a/63156174/7313822) that explains it well (to me at least):
> viewBox is the difference between your SVG being responsive and not.
>
> With a viewBox if you rescale your canvas, the contents will rescale too, without it the contents will stay the same size and you'll either be able to see more or less of it kind of like putting a picture frame over a picture.

Comment (slothtown) [reply img/c/cmore.se.svg]: Thanks, I'll keep that in mind in the future!

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**